Source/WebCore/Modules/mediastream/gstreamer/GStreamerMediaEndpoint.cpp

```diff
@@ -312,6 +312,16 @@ static std::optional<std::pair<RTCSdpType, String>> fetchDescription(GstElement*
     if (!description)
         return { };
 
+    unsigned i = 0;
+    while (i < gst_sdp_message_attributes_len(description->sdp)) {
+        const auto attribute = gst_sdp_message_get_attribute(description->sdp, i);
+        if (!g_strcmp0(attribute->key, "end-of-candidates")) {
+            gst_sdp_message_remove_attribute(description->sdp, i);
+            continue;
```

Comment:
Shouldn't this be a break? 

If you have to remove more  than one (which does not make sense to me) you would be doing the same iteration twice, meaning you find the first, you remove, you don't increate `i`, you iterate another time with the same `i`, you don't find it, you increase the `i` and next iteration. So if are in this case, I would directly do `while (i++ ...`.

Reply:
Right, let's make this a `break`.